core/src/main/java/org/elasticsearch/ingest/PipelineStore.java

```diff
@@ -51,6 +51,7 @@
 
     private final Pipeline.Factory factory = new Pipeline.Factory();
     private final Map<String, Processor.Factory> processorFactories;
+    private final boolean newDateFormat;
```

Comment:
will do, guess I took it for granted that this resides in an ingest-specific class.